webapp/channels/src/components/plugin_link_tooltip/index.tsx

```diff
@@ -61,11 +61,15 @@ export default function PluginLinkTooltip(props: Props) {
         dismissInteraction,
     ]);
 
+    const nodeAttributes = useMemo(() => {
+        return convertPropsToReactStandard(props.nodeAttributes);
```

Comment:
I'm surprised that we haven't run into this with any of the other components rendered by `messageHtmlToComponent`. I guess they either don't use `replaceChildren: false` or they don't pass a class name through there.

This seems like it should be part of `messageHtmlToComponent` since it's what's giving us the incorrectly-named props. Would you mind moving this to there? That way, the type of `props` for this component should actually be correct

Reply:
done